main.py

```diff
@@ -671,14 +671,36 @@ def batch_process(users_chunk, batch_args):
         repos_with_tags_sizes=repo_sizes,
         total_tags=len(tags),
         pull_push_batch_size=env_config["batch_size"],
-        number_of_push_pull_jobs_per_user=len(tags)//env_config["batch_size"],
     )
 
     namespace = env_config["test_namespace"]
 
-    if not ({'load', 'run', 'delete'} & set(phases_list)):
-        logging.info("No valid phases defined to run the tests. Valid options: LOAD, RUN and DELETE")
+    if not ({'load', 'run', 'delete', 'push_pull'} & set(phases_list)):
+        logging.info("No valid phases defined to run the tests. Valid options: LOAD, RUN, PUSH_PULL and DELETE")
         sys.exit()
+    
+    batch_args = {
+    "namespace": namespace,
+    "quay_host": env_config["quay_host"],
+    "concurrency": env_config["concurrency"],
+    "uuid": env_config["test_uuid"],
+    "auth_token": env_config["auth_token"],
+    "batch_size": env_config["batch_size"],
+    "tags": tags,
```

Comment:
Could we pass in a set of tags to pull explicitly via environment variables? i.e. have something like `TAGS="registry.stage.redhat.io/ubi9/ubi,registry.stage.redhat.io/ubi9/podman"`. It looks like the tags are based on what the tests can push to quay, but for the registry proxy it won't be able to push any images

Reply:
This ask seems very specific to proxy. I would say, use this PR to upload images without proxy and then pull them using a custom loop script (which should be pretty simple to write) by enabling the proxy if that makes sense